src/main/java/org/broadinstitute/hellbender/tools/spark/sv/Kmer.java

```diff
@@ -0,0 +1,177 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import java.io.Serializable;
+
+/**
+ * An immutable Kmer.
+ * K must be between 1 and 63 (but it's silly to use this class for K < 33).
+ * Canonicalization is unimplemented for even K.
+ *
+ * Created by tsharpe on 12/10/15.
```

Comment:
We usually don't add name tags to things since git captures it.  It's ok to leave it, but it's non-standard.